sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/tests/TimeSeriesInsightsInstancesTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Azure.Iot.TimeSeriesInsights.Tests
+{
+    public class TimeSeriesInsightsInstancesTests : E2eTestBase
+    {
+        // TODO: replace hardcoding the Type GUID when the Types resource has been implemented
+        private const string DefaultType = "1be09af9-f089-4d6b-9f0b-48018b5f7393";
+        private const int MaxNumberOfRetries = 10;
+
+        private readonly TimeSpan _delay = TimeSpan.FromSeconds(10);
```

Comment:
`_delay` is a non-descript field name. Can we be more specific? Delay of what?

Reply:
will rename to _retryDelay